apps/cookbook/src/app/examples/chart-example/examples/monthly-overview-chart-example/monthly-overview-chart-example.component.ts

```diff
@@ -0,0 +1,189 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Options, SeriesClickCallbackFunction, SeriesClickEventObject } from 'highcharts';
+
+import { ModalController, DesignTokenHelper, ChartComponent } from '@kirbydesign/designsystem';
+
+const getColor = DesignTokenHelper.getColor;
+const fontSize = DesignTokenHelper.fontSize;
+
+function colorPoints(selectedIdx) {
+  return function() {
+    var series = this.series;
+    for (var i = 0, ie = series.length; i < ie; ++i) {
+      var points = series[i].data;
+      for (var j = 0, je = points.length; j < je; ++j) {
+        if (points[j].graphic) {
+          if (j === selectedIdx) {
+            points[j].graphic.element.style.fill = getColor('primary').value;
+            points[j].graphic.element.style.stroke = getColor('primary').value;
+          } else {
+            points[j].graphic.element.style.stroke = getColor('secondary').value;
+          }
+        }
+      }
+    }
+  };
+}
+
+@Component({
+  selector: 'cookbook-monthly-overview-chart-example',
```

Comment:
Please align examples naming with existing examples:
```suggestion
  selector: 'cookbook-chart-example-monthly-overview',
```